docs/03-tutorials/api-exposure/apix-04-own-domain.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Use a custom domain to expose a service
+---
+
+This tutorial shows how to set up your custom domain and prepare a certificate to use the domain for exposing a service. The components used are Gardener [External DNS Management](https://gardener.cloud/docs/concepts/networking/dns-managment/#external-dns-management) and [Certificate Management](https://gardener.cloud/docs/concepts/networking/cert-managment/).
+
+To learn how to expose a service, go to [this](./apix-01-expose-service-apigateway.md) tutorial.
+
+For this tutorial, use Kyma 2.0 or higher.
+
+## Prerequisites
+
+If you use a cluster not managed by Gardener, install the required components manually. Follow these steps:
+
+1. Create a Namespace. Run:
+
+   ```bash
+   kubectl create ns gardener-components
+   ```
+
+2. Download the [External DNS Management](https://github.com/gardener/external-dns-management) and [Certificate Management](https://github.com/gardener/cert-management) projects locally. Run:
+
+   ```bash
+   git clone --branch v0.10.4 https://github.com/gardener/external-dns-management.git
+   git clone --branch v0.8.3 https://github.com/gardener/cert-management.git
+   ```
+
+   > **NOTE:** This tutorial uses the External DNS Management v.0.10.4 and the Certificate Management v0.8.3.
+
+3. Enable vertical Pod autoscaling on your cluster. For example, for Google Cloud Platform, run:
+
+   ```bash
+   gcloud container clusters update {CLUSTER_NAME} --enable-vertical-pod-autoscaling --zone="{CLUSTER_ZONE}" --project="{PROJECT_NAME}"
+   ```
+
+4. Install the `external-dns-management` component in the `gardener-components` Namespace:
+
+   ```bash
+   helm install external-dns {PATH_TO_EXTERNAL_DNS_MANAGEMENT}/charts/external-dns-management --namespace=gardener-components --set configuration.identifier=external-dns-identifier
+   ```
+
+5. Install the `cert-management` component in the `gardener-components` Namespace:
+
+   ```bash
+   helm install cert-manager {PATH_TO_CERT_MANAGEMENT}/charts/cert-management --namespace=gardener-components --set configuration.identifier=cert-manager-identifier
+   ```
+
+## Steps
+
+Follow these steps to set up your custom domain and prepare a certificate required to expose a service.
+
+1. Install Kyma 2.0 or higher.
```

Comment:
It's so obvious to us, we didn't notice: Users would prefer  a copy-paste command here or a link to docu.